packages/core/src/command/consolidated/consolidated-create.ts

```diff
@@ -132,28 +137,43 @@ export async function createConsolidatedFromConversions({
   await dangerouslyDeduplicate({ cxId, patientId, bundle });
   log(`...done, from ${lengthWithDups} to ${bundle.entry?.length} resources`);
 
+  log(`Normalizing consolidated bundle...`);
+  const normalizedBundle = normalizeFhir(bundle);
```

Comment:
This is cool, and have been mapped on https://linear.app/metriport/issue/ENG-316. Can you take a look on todos for that (eng-316), please? There's one place we should remove the call to `normalize()` from.

Also, let's call `normalize()`, not `normalizeFhir()`, please.

Reply:
Let me know if this was added correctly - I haven't looked into `normalize` as much as the other bundle operations.